runtime/runtime-params-estimator/src/gas_cost.rs

```diff
@@ -76,6 +76,21 @@ impl GasCost {
         result
     }
 
+    /// Like cmp::Ord::min but operates on heterogenous types (GasCost + Gas).
```

Comment:
```suggestion
    /// Like [`std::cmp::Ord::min`] but operates on heterogenous types ([`GasCost`] + [`Gas`]).
```

Will auto-link the relevant parts of the documentation.